features/support/env.rb

```diff
@@ -6,25 +6,16 @@
 require 'rspec'
 
 
-SOURCE_DIRECTORY = "#{File.dirname(__FILE__)}/../../src"
+SOURCE_DIRECTORY = File.join(File.dirname(__FILE__), '..', '..', 'src')
 GIT_TOWN_DIRECTORY = File.expand_path('..', SOURCE_DIRECTORY)
-SHELL_OVERRIDE_DIRECTORY = "#{File.dirname(__FILE__)}/shell_overrides"
+SHELL_OVERRIDE_DIRECTORY = File.join(File.dirname(__FILE__), 'shell_overrides')
 
 MEMOIZED_REPOSITORY_BASE = Dir.mktmpdir 'memoized'
 REPOSITORY_BASE = Dir.mktmpdir
-TOOLS_INSTALLED_FILENAME = "#{REPOSITORY_BASE}/tools_installed.txt"
+TOOLS_INSTALLED_FILENAME = File.join(REPOSITORY_BASE, 'tools_installed.txt')
 
 FISH_AUTOCOMPLETIONS_PATH = File.expand_path '~/.config/fish/completions/git.fish'
 
-DEBUG = {
-
-  # Prints debug info for all activities
-  all: ENV['DEBUG'],
-
-  # Prints debug info only for the Git commands run
-  commands_only: ENV['DEBUG_COMMANDS']
-}
-
```

Comment:
Rubocop wanted this frozen since its supposedly a constant. Changed how it works.